app/Models/User.php

```diff
@@ -57,7 +57,7 @@ public static function isAdmin($roleId)
      *
      * @param $userId
      *
-     * @return boolean
+     * @return bool
```

Comment:
@vuchl It would be a `bool` since the actual value that is being returned is either `true` or `false`.